demo/admin/src/products/future/ProductsGrid.cometGen.ts

```diff
@@ -14,21 +16,51 @@ export const ProductsGrid: GridConfig<GQLProduct> = {
             name: "overview",
             headerName: "Overview",
             minWidth: 200,
-            maxWidth: 250,
             primaryText: "title",
-            secondaryText: "description",
+            secondaryText: {
+                type: "formattedMessage",
+                message: "{price} • {type} • {category} • {inStock}",
```

Comment:
We don't want to use this for separators, so we shouldn't have an example in Demo for this IMO.

Reply:
Since there currently is no other way of achieving this I simply added a TODO comment for the follow-up PR. 
Is that okay with you?

5270b86ede769859d2d5726072f3ec09308b07ee

Reply:
Changed to the "group" type in https://github.com/vivid-planet/comet/pull/2627